src/persist/src/operators/stream.rs

```diff
@@ -721,6 +694,77 @@ mod tests {
         Ok(())
     }
 
+    /// Test to make sure we handle closing the seal operator correctly and don't
+    /// incorrectly process any seal futures after the operator has been closed.
+    #[test]
+    fn seal_close() -> Result<(), Error> {
```

Comment:
this is a regression test for 9419 right? let's name it like the other regression tests